README.md

```diff
@@ -44,9 +44,7 @@ bot
 
     const channel = {name: 'kbot,' + bot.myInfo().username, public: false, topicType: 'chat'}
     const message = {
-      body: `Hello kbot! This is ${bot.myInfo().username} saying hello from my device ${
```

Comment:
These changes are from prettier, due to the fact that we increased the line width to 140 characters.